alibi/explainers/anchor_base.py

```diff
@@ -835,7 +836,7 @@ def __init__(self, samplers: List[Callable], **kwargs) -> None:
         self.pool = ActorPool(samplers)
         self.samplers = samplers
 
-    def _get_coverage_samples(self, coverage_samples: int, samplers: List[Callable] = None) -> np.ndarray:
+    def _get_coverage_samples(self, coverage_samples: int, samplers: List[Callable]) -> np.ndarray:  # type: ignore
```

Comment:
It's because this is an inherited class that overrides the parent method, but the parent methods has `samplers` as `Optional` whereas here it is required, this results in a typing error. I will see if I can narrow down the ignores in this PR to more specific error codes.